src/Pickles/Pickles.ObjectModel/ObjectModel/Step.cs

```diff
@@ -31,5 +38,9 @@ public class Step
         public Table TableArgument { get; set; }
 
         public string DocStringArgument { get; set; }
+
+        public Location Location { get; set; }
+
+        public List<Comment> Comments { get; set; } 
```

Comment:
Please remove the extra whitespace at the end of the `Comments { get; set; }` line